tests/runtime/test_grpc_server.py

```diff
@@ -1283,6 +1283,25 @@ def test_construct_with_options(open_port, sample_train_service, sample_int_file
             assert context.value.code() == grpc.StatusCode.RESOURCE_EXHAUSTED
 
 
+def test_grpc_server_startup(open_port, sample_train_service, sample_int_file):
+    """Make sure that the server can be booted with config options"""
```

Comment:
"...booted with config options to enable listening on a socket"?

Reply:
🤦 thank you for catching me going to fast. I also updated the function name to be more descriptive